src/Utilities/Utilities.jl

```diff
@@ -114,11 +114,11 @@ parsed from.
 The keyword argument `skip = N` drops the leading `N` lines from the input string.
 """
 function parseblock(code::AbstractString, doc, page; skip = 0, keywords = true)
+    # work only with LF line endings internally
+    code = replace(code, "\r\n" => "\n")
```

Comment:
Actually, this could be dangerous if someone have `"\r\n"` in a string. Should revert this and let the Julia parser decide this kind of things instead.